packages/core/src/networks.ts

```diff
@@ -122,11 +124,85 @@ export const fetchDripVersionForNetwork = (chainId: bigint) => {
   }
 }
 
-export const isVerificationSupportedForNetwork = (chainId: bigint) => {
-  const network = SPHINX_NETWORKS.find((n) => n.chainId === chainId)
+export const isEtherscanSupportedForNetwork = (
+  chainId: bigint,
+  mockSphinxNetworks?: any
+) => {
+  const networks: Array<SupportedNetwork> =
+    mockSphinxNetworks ?? SPHINX_NETWORKS
+  const network = networks.find((n) => n.chainId === chainId)
+
+  if (network) {
+    return network.blockexplorers.etherscan !== undefined
+  } else {
+    throw new Error(`Unsupported network id ${chainId}`)
+  }
+}
+
+export const isBlockscoutSupportedForNetwork = (
+  chainId: bigint,
+  mockSphinxNetworks?: any
+) => {
+  const networks: Array<SupportedNetwork> =
+    mockSphinxNetworks ?? SPHINX_NETWORKS
+  const network = networks.find((n) => n.chainId === chainId)
+
+  if (network) {
+    return network.blockexplorers.blockscout !== undefined
+  } else {
+    throw new Error(`Unsupported network id ${chainId}`)
+  }
+}
+
+export const isVerificationSupportedForNetwork = (
+  chainId: bigint,
+  mockSphinxNetworks?: any
+) => {
+  return (
+    isEtherscanSupportedForNetwork(chainId, mockSphinxNetworks) ||
+    isBlockscoutSupportedForNetwork(chainId, mockSphinxNetworks)
+  )
+}
+
+export const fetchEtherscanConfigForNetwork = (
```

Comment:
This is a surprisingly complex function. You can look at the test file below for a good overview of the different cases it covers. 

There are basically two cases this function is useful for:
1. We have a specific block explorer, and we want the config for exactly that block explorer
2. We don't know what block explorers are available on the network and just want to get the first explorer available, defaulting to etherscan if more than one is available.

We could have used two separate functions for this, but then we'd have to change the verifySphinxConfig function to handle that and then we'd end up with even less code shared between the website and deploy command. So I decided to just do this and make sure the function is well tested.